package.json

```diff
@@ -5,7 +5,7 @@
   "main": "dist/rollup.js",
   "module": "dist/rollup.es.js",
   "jsnext:main": "dist/rollup.es.js",
-  "typings": "dist/typings/node-entry.d.ts",
+  "typings": "dist/rollup.d.ts",
```

Comment:
Since there are no files in `dist/typings` and the files in `typings` are not relevant anymore for the published artifact:
1. The script `"pretest:typescript"` can be simplified to `"rm -rf test/typescript/dist && cp -r dist test/typescript/"` and
2. `"typings/package.json.d.ts"` can be removed from `"files"`.